work_queue/src/work_queue_process.c

```diff
@@ -256,6 +322,19 @@ pid_t work_queue_process_execute(struct work_queue_process *p )
 		if(result == -1)
 			fatal("could not dup pipe to stderr: %s", strerror(errno));
 
+		if(strcmp(p->task->command_line, "@FUNCTION") == 0) {	
```

Comment:
This makes me a little nervous. I think having special command lines is ok for testing, but current users would expect to "opt-in", as it is possible that somebody does have a "@FUNCTION" executable.

I suggest adding a  --python-function argument to the work queue worker which will keep the command_line to check against, e.g.  work_queue_worker `--python-function @my_func`.   In the future we will then execute some_module.my_func.